CHANGELOG.latest.md

```diff
@@ -1,82 +1,4 @@
-
-### Migration Guides
-- See [Android Native - 5.x to 6.x Migration](https://www.revenuecat.com/docs/android-native-5x-to-6x-migration) for a
-  more thorough explanation of the new Google subscription model announced with BillingClient 5 and how to take
-  advantage of it in V6. This guide includes tips on product setup with the new model.
-
-### New `SubscriptionOption` concept
-
-#### Purchasing
-In v4, a Google Play Android `Package` or `StoreProduct` represented a single purchaseable entity, and free trials or intro
-offers would automatically be applied to the purchase if the user was eligible.
-
-Now, in Hybrid Common v5, an Google Play Android `Package` or `StoreProduct` represents a duration of a subscription and contains all the ways to
-purchase that duration -- any offers and its base plan. Each of these purchase options are `SubscriptionOption`s.
-When passing a `Package` to `purchasePackage()` or `StoreProduct` to `purchaseStoreProduct()`, the SDK will use the following logic to choose which
-`SubscriptionOption` to purchase:
-- Filters out offers with "rc-ignore-offer" tag
-- Uses `SubscriptionOption` with the longest free trial or cheapest first phase
-    - Only offers the user is eligible will be applied
-- Falls back to base plan
-
-For more control, purchase subscription options with the new `purchaseSubscriptionOption()` method.
-
-#### Models
-
-`StoreProduct` now has a few new properties use for Google Play Android:
-- `defaultOption`
-  - A subscription option that will automatically be applie when purchasing a `Package` or `StoreProduct`
-- `subscriptionOptions`
-  - A list of subscription options (could be null)
-
-##### Subscription Option
-
-Below is an example of what a subscription option:
-
-```js
-{
-    "id": "basePlan",
-    "storeProductId": "subId:basePlanId",
-    "productId": "subId",
-    "pricingPhases": [
-        {
-            "price": 0,
-            "priceString": "FREE",
-            "period": "P1M",
-            "cycles": 1
-        },
-        {
-            "price": 4.99,
-            "priceString": "$4.99",
-            "period": "P1M",
-            "cycles": 0
-        }
-    ],
-    "tags": ["free-offers"],
-    "isBasePlan": false,
-    "billingPeriod": {
-        "periodUnit": "MONTH",
-        "periodNumberOfUnits": 0
-    },
-    "fullPricePhase": {
-        "price": 4.99,
-        "priceString": "$4.99",
-        "period": "P1M",
-        "cycles": 0
-    },
-    "freePhase" {
-        "price": 0,
-        "priceString": "FREE",
-        "period": "P1M",
-        "cycles": 1
-    },
-    "introPhase": null
-}
-```
-
-### Observer Mode
-
-Observer mode is still supported in v5. Other than updating the SDK version, there are no changes required.
+### Breaking Changes
+* [BC5] Rename `Period` fields to `unit` and `value` (#365) via Josh Holtz (@joshdholtz)
+### New Features
```

Comment:
maybe we can rename this to "bug fixes"? in case someone had issues with the first one..

Reply:
Done!